### ISSUE TITLE
Embedded Todos (can't get to work)

### ISSUE THREAD

**Issue by mtom55** [fabiospampinato/vscode-todo-plus#308]
I can't for the life of me work out how to get embedded todo's to work. I've reread the readme multiple times.

So far my best attempt has been:
1. brew install the_silver_searcher
2. Preferences > Extensions > Todo Plus >  Settings > Edit Settings in Json
3. add
```js

"settings": {
		"todo.embedded.provider": "ag",
		"todoPlus.languages": [
		]
	}
```
**Expected:**
to see todo items showing up on the todo sidebar (under incomplete)

**Actual:**
Incomplete and Complete are empty

**Comment by fabiospampinato** [comment]
@mtom55 is this still a problem in v4.18.3?

**Comment by mtom55** [comment]
That's all working thankyou very much!  Fixed in 14.8.3
